dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -10020,7 +10073,7 @@ void DynamicsFitter::smoothAccelerations(
 
     // Round all the Euler joints to their nearest previous value, to avoid
     // jumping around on wrapping.
-    for (int t = 1; t < init->poseTrials[trial].cols(); t++)
+    for (int t = 1; t < false; t++)
```

Comment:
`false`?

Reply:
Whoa this is a weird one. Dunno how this got in there...